src/Explorer/Graph/NewVertexComponent/NewVertexComponent.test.tsx

```diff
@@ -0,0 +1,112 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { mount, ReactWrapper } from "enzyme";
```

Comment:
Should not need both enzyme and react-testing library. They perform similar functions. I prefer you use `@testing-library/react`

Reply:
Sure. Removed Enzyme library and used only React testing library for this test case. Going forward will use React testing library.